src/OpenTelemetry/Resources/Resource.cs

```diff
@@ -57,34 +57,34 @@ internal Resource(IEnumerable<KeyValuePair<string, object>> attributes)
         public IEnumerable<KeyValuePair<string, object>> Attributes { get; }
 
         /// <summary>
-        /// Returns a new, merged <see cref="Resource"/> by merging the current <see cref="Resource"/> with the.
-        /// <code>other</code> <see cref="Resource"/>. In case of a collision the current <see cref="Resource"/> takes precedence.
+        /// Returns a new, merged <see cref="Resource"/> by merging the old <see cref="Resource"/> with the.
```

Comment:
nit: Looks like there's a period at the end of the first line that should be removed.

Reply:
good catch, seems error SA1629 prompts you to add a period to the end of documentation text.

Reply:
I think the issue is the `<code>` which creates a line break. Try switching it to `<c>other</c>` which will render inline.